docs/content/latest/benchmark/tpcc-ysql/_index.md

```diff
@@ -63,7 +74,15 @@ Other options like username, password, port, etc. can be changed using the confi
 <password></password>
 ```
 
-## 3. Run the TPC-C benchmark
+## 3. Best practices
+
+**Latest TPCC code:** Use the latest enhancements to the Yugabyte TPCC application `git clone`.
```

Comment:
Is this a useful change? It's at least two complete sentences, which seems like an improvement ;-)
```suggestion
**Latest TPCC code:** Use the latest enhancements to the Yugabyte TPCC application. You can either download the latest released version, or, to get the very latest changes, you can clone the repository and build from source.
```